src/Tribe/Assets.php

```diff
@@ -75,6 +75,22 @@ public function enqueue_scripts() {
 						'tribe-tickets-rsvp',
 						'tribe-tickets-registration-page',
 					],
+				],
+			);
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
```

Comment:
:no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: a346e844fab2a992c36e05f56b15b468</sub>